src/firefly/js/ui/FileUploadProcessor.jsx

```diff
@@ -49,31 +49,39 @@ const tableWarning = 'Only loading the image(s), ignoring any selected table(s).
 
 export function resultSuccess(request) {
 
-    const {currentReport, currentDetailsModel, summaryModel, groupKey, acceptList, uniqueTypes}= request.additionalParams ?? {};
+    const {report, detailsModel, summaryModel, groupKey, acceptList, uniqueTypes,
```

Comment:
I like `report` better than `currentReport` and `detailsModel` better than `currentDetailsModel`